cli/repl.js

```diff
@@ -0,0 +1,27 @@
+var repl = require('repl')
+
+var AccountAdmin = require('@hoodie/account-client/admin')
+
+var Loader = require('./loader')
+var getCliOptions = require('./options.js');
+
+var options = getCliOptions(/*how to get the proct path???*/)
```

Comment:
`process.cwd()` is what you are looking for, see https://github.com/hoodiehq/hoodie/blob/be9e92fa700cc028d3c4c469f595963cfa4f1bca/cli/index.js#L20-L21